### PR TITLE
update containerless doc &  java provider 

### PR DESCRIPTION
**Pull request by eemcmullan**

### CODE REVIEW

File: cmd/analyze-bin.go [hunk 1]
```diff
@@ -187,7 +187,7 @@ func NewAnalyzeBinCmd(log logr.Logger) *cobra.Command {
 			needProviders := map[string]provider.InternalProviderClient{}
 
 			if analyzeBinCmd.enableDefaultRulesets {
-				analyzeBinCmd.rules = append(analyzeBinCmd.rules, filepath.Join(analyzeBinCmd.homeKantraDir, RulesetsLocation))
+				analyzeBinCmd.rules = append(analyzeBinCmd.rules, filepath.Join(analyzeBinCmd.homeKantraDir, "rulesets"))
```
Comment (dymurray) [inline cmd/analyze-bin.go:190]: Just change the constant value to "rulesets"

Comment (dymurray) [reply cmd/analyze-bin.go:190]: https://github.com/konveyor/kantra/pull/347

if you just want to merge and rebase it

**Comment by dymurray** [comment]
This worked for me :+1: just fix the const and we are good

**Review by aufi — APPROVED** [review]
LGTM

**Review by dymurray — APPROVED** [review]

**Pull request merged**